OrbitGl/App.cpp

```diff
@@ -503,6 +511,7 @@ void OrbitApp::MainTick() {
 
 void OrbitApp::RegisterCaptureWindow(CaptureWindow* capture) {
```

Comment:
Nit: You obviously didn't introduce this, but I find the function names `RegisteCaptureWindow` (and the other `RegisterXXX` below) confusing - to me, this signals you can register more than one capture window here, when in fact the CHECK at the beginning demands the exact opposite.

Would just calling it `SetCaptureWindow` be clearer?

Reply:
Totally agree, thanks, done.